homeassistant/components/knx/__init__.py

```diff
@@ -148,6 +148,11 @@ async def async_setup(hass, config):
             discovery.async_load_platform(hass, platform.value, DOMAIN, {}, config)
         )
 
+    if len(hass.data[DATA_KNX].xknx.devices) == 0:
+        _LOGGER.warning(
+            "No KNX devices are configured. Please make sure to migrate your configuration according to https://xknx.io/migration_ha_0115.html."
```

Comment:
Please instead link to the breaking change section on our release page where the change is described. That section should also have a link to the home assistant knx integration docs.

Reply:
Also please break the string around 88 characters.